source/user_guide.rst

```diff
@@ -9,20 +9,95 @@ platform itself, please refer to the [operations guide]()
 .. contents:: Table of Contents
   :local:
 
+Who should read this
+--------------------
+
+- Users that want a deeper look at how dojot works;
+- Application developers
+
+
 Getting Started
 ---------------
 
-The idea here is to present the reader with a sample end-to-end usage of the platform, showcasing
-a relevant set of the features implemented by the platform, thus serving as an example of how to
-use the platform on a "real world" scenario.
+To start, please follow dojot's installation guide. There you should find how to properly download a working copy of the components, how to minimally configure them, how to start them up and how to check whether they are working.
+
+dojot basics
+------------
+
+Before using dojot, the user must be familiar with some basic operations and concepts. They are very simple to understand and use, but without them all operations might become obscure and senseless. It is advisable to checkout our `architecture description <architecture.html>`_ to get acquainted with all internal components.
+
+
+User authentication
+*******************
+
+All HTTP requests supported by dojot are sent to the API gateway. In order to control which user should access which endpoints and resources, dojot makes uses of JWT (JSON Web Tokens) which encodes things like (not limited to these):
+
+- User identity
+- Validation data
+- Timestamp
+
+The component responsible for user authentication is `auth <https://github.com/dojot/auth>`_.
+
+
+Devices and templates
+*********************
+
+In dojot, a device is a digital representation of an actual device or gateway with one or more sensors or of a virtual one with sensors/attributes inferred from other devices. Throughout the documentation, this kind of device will be called simply as 'device'. If the actual device must be referenced, we'll be calling it as 'physical device'.
+
+Consider, for instance, an actual device with temperature and humidity sensors; it can be represented into dojot as a device with two attributes (one for each sensor). We call this kind of device as regular device or by its communication protocol, for instance, MQTT device or CoAP device.
```

Comment:
> Consider, for instance, an actual device with temperature

physical device, as stated on the previous paragraph

Reply:
Okok...